### ISSUE TITLE
Replace all the sensitive/configurable settings in the django project with environment variables

### ISSUE THREAD

**Issue by napulen** [DDMAL/cantus#298]
We used to require that whoever deploys a new instance of the website manually-modifies the django settings by editing the `example-settings.py` file and saving it into `settings.py`. This is time-consuming and prone to errors.

Now, we only have a `settings.py` file, which includes all the necessary configurations necessary for the project to work within the container. There is still the issue of having sensitive/configurable information (e.g., passwords, ports, usernames) as plain-text in this `settings.py` file. This should be addressed by making the `settings.py` file access to this information through environment variables instead. Those environment variables can be defined in the staging/production environment by whoever launches the container. It is a better solution for automation and less prone to errors than manually modifying files every time the website is launched somewhere.

For now, the `settings.py` file has placeholders instead of environment variables. It is a low priority to change this during the development of the container and a **high priority** once the containerization is ready.
